rviz_default_plugins/src/rviz_default_plugins/tools/pose_estimate/initial_pose_tool.cpp

```diff
@@ -95,9 +106,9 @@ void InitialPoseTool::onPoseSet(double x, double y, double theta)
 
   pose.pose.pose.orientation = orientationAroundZAxis(theta);
 
-  pose.pose.covariance[6 * 0 + 0] = 0.5 * 0.5;
-  pose.pose.covariance[6 * 1 + 1] = 0.5 * 0.5;
-  pose.pose.covariance[6 * 5 + 5] = M_PI / 12.0 * M_PI / 12.0;
+  pose.pose.covariance[6 * 0 + 0] = covariance_x_property_->getFloat();
```

Comment:
Not sure what the conventions are, either one is fine with me. What are the conventions @dirk-thomas @wjwwood ?

Reply:
The covariance of two identical variables is the same as that variables variance (which is the standard deviation squared). So, I think the hard-coded values here represent the standard deviation for each variable. Since we're naming the property "covariance", I don't think we should square the value provided by the user.